pkgs/development/web/ihp-new/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, git, makeWrapper }:
+stdenv.mkDerivation rec {
+  pname = "ihp-new";
+  version = "0.20.0";
+
+  src = fetchFromGitHub {
+    owner = "digitallyinduced";
+    repo = "ihp";
+    rev = "b7f856d982d7e82390f79d109c1f58ef2a365886";
+    sha256 = "sha256-jcuY1bZfpLpJYhh3UQCBEF856O8AT/R6hQjy2cYJhEw";
+  };
+
+  dontConfigure = true;
+  sourceRoot = "source/ProjectGenerator";
+  buildInputs = [ git makeWrapper ];
+  installPhase = ''
+    mkdir -p $out/bin;
+    cp bin/ihp-new $out;
+    makeWrapper $out/ihp-new $out/bin/ihp-new --prefix PATH ":" "${git}/bin";
+  '';
+
+  meta = with lib; {
+    description = "Project generator for the IHP (Integrated Haskell Platform) web framework";
+    license = licenses.mit;
+    maintainers = [ maintainers.mpscholten ];
+    platforms = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

```suggestion
    platforms = platforms.unix;
```

basically unix and good enough for hydra.